drush/Commands/SiteInstallCommands.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Drush\Commands;
+
+use Consolidation\AnnotatedCommand\CommandData;
+use Consolidation\AnnotatedCommand\CommandResult;
+use Drupal\cohesion\Drush\DX8CommandHelpers;
+
+/**
+ * Rebuild site studio when site is installed via drush.
+ */
+class SiteInstallCommands extends DrushCommands {
+
+  /**
+   * Do a forceful rebuild whenever site is installed.
+   *
+   * @hook post-command site-install
+   */
+  public function postCommand($result, CommandData $commandData) {
+    $arguments = $commandData->arguments();
+    $moduleHandler = \Drupal::service('module_handler');
+    if (isset ($arguments['profile'][0]) && $arguments['profile'][0] == 'acquia_cms' && $moduleHandler->moduleExists('cohesion')) {
```

Comment:
@saurabhtripathics - there shouldn't be a space after `isset` - should be `isset()`, right?

Reply:
fixed